scripts/rook.sh

```diff
@@ -8,6 +8,22 @@ ROOK_CEPH_CLUSTER_VERSION="v14.2.10"
 KUBECTL_RETRY=5
 KUBECTL_RETRY_DELAY=10
 
+trap log_errors EXIT
+
+# log_errors is called on exit (see 'trap' above) and tries to provide
+# sufficient information to debug deployment problems
+function log_errors() {
+	# enable verbose execution, disable exit on failure
+	set -x +e
+	kubectl get nodes
+	kubectl -n rook-ceph get events
+	kubectl -n rook-ceph get pods --show-labels
```

Comment:
pod describe the output of pods in the rook-ceph namespace also helps

Reply:
sure, I can do that instead